parts/kubernetesagentcustomdata.yml

```diff
@@ -70,7 +70,7 @@ write_files:
     - name: localcluster
       cluster:
         certificate-authority: /etc/kubernetes/certs/ca.crt
-        server: https://{{{masterPrivateIp}}}:443
+        server: https://{{{masterInternalLbIp}}}:443
```

Comment:
did you search for all occurences of "masterPrivateIP" in the Yaml and templates file?  I don't see a change to the windows agents anywhere in here.